daffodil-propgen/src/main/resources/org/apache/daffodil/xsd/dafext.xsd

```diff
@@ -110,55 +110,39 @@
             </xs:documentation>
           </xs:annotation>
         </xs:element>
-        <xs:element name="blobChunkSizeInBytes" type="xs:int" default="4096" minOccurs="0">
+        <xs:element name="blobChunkSizeInBytes" default="4096" minOccurs="0">
           <xs:annotation>
             <xs:documentation>
               When reading/writing blob data, the maximum number of bytes to read/write at a time.
             </xs:documentation>
           </xs:annotation>
+          <xs:simpleType>
```

Comment:
Your scraper generator depends on this being an immediate simpletype with a restriction with min/max/exclusive/inclusive in it manifestly. 

You need to document that here, because changing this to say type="daf:oneOrGreater" with that defined elsewhere would break this I think.